### PR TITLE
Support reverseOpenLR property for bidirectional ways

### PR DESCRIPTION
**Pull request by nickcordella**
this adds support for a reverseOpenLR property that is useful for bidirectional ways.

### CODE REVIEW

File: test/fixtures/normalize-ways/12022001101.json [hunk 1]
```diff
@@ -1,3 +1,3 @@
...
-{"type": "Feature", "properties": {"oneway": "1", "refs": ["11", "22", "33"], "id": "223|0!0;12022001101", "highway": "motorway", "openLR": "CwG5MSK/4QHkNg2uBp4BBQ=="}, "geometry": {"type":"LineString", "coordinates": [[2.4235153198242188,48.866521538507754],[2.4451446533203125,48.877812415009195],[2.4585342407226562,48.883456897746534]]}}
-{"type": "Feature", "properties": {"oneway": "0", "refs": ["44", "55"], "id": "224|0!0;12022001101", "highway": "residential", "openLR": "CwG/kSLC9yj/Av/TAJcoHw=="}, "geometry": {"type":"LineString", "coordinates": [[2.4585342407226562,48.883456897746534],[2.4580836296081543,48.884966688891]]}}
+{"type": "Feature", "properties": {"oneway": 1, "refs": ["1", "2", "3"], "id": "123|0!0;12022001101", "highway": "motorway", "openLR": "CwG/kSLC9wH1NvJS+WIBFA=="}, "geometry": {"type":"LineString", "coordinates": [[2.4585342407226562,48.883456897746534],[2.4451446533203125,48.877812415009195],[2.4235153198242188,48.866521538507754]]}}
```
Comment (camilosanin) [inline test/fixtures/normalize-ways/12022001101.json]: why did you have to make these numbers? what is the usual input?

File: bin/normalize-ways [hunk 2]
```diff
@@ -78,13 +78,19 @@ waysStream.on('data', function (line) {
...
+        }
       }
       outputStream.write(JSON.stringify(way) + '\n');
     }
     way = ways[ways.length - 1];
     way.properties.id = way.properties.id + ';' + quadkey;
     if (openLR) {
       way.properties.openLR = geojsonLR.toOpenLRString(way);
+      if (way.properties.oneway === 0) {
+        way.properties.reverseOpenLR = geojsonLR.reverseOpenLRString(way.properties.openLR);
```
Comment (camilosanin) [inline bin/normalize-ways]: 😩

File: bin/normalize-ways [hunk 3]
```diff
@@ -78,13 +78,19 @@ waysStream.on('data', function (line) {
       way.properties.id = way.properties.id + ';' + quadkey;
       if (openLR) {
         way.properties.openLR = geojsonLR.toOpenLRString(way);
+        if (way.properties.oneway === 0) {
```
Comment (camilosanin) [inline bin/normalize-ways]: would it be better if we make `+way.properties.oneway === 0` or `String(way.properties.oneway) === "0"` ?

File: test/fixtures/normalize-ways/12022001101.json (hunk 1 above)
Comment (nickcordella) [reply test/fixtures/normalize-ways/12022001101.json]: the fixtures were actually invalid to begin with - graph-normalizer expects integer values here.

**Pull request merged**